doc/whats_new/v1.0.rst

```diff
@@ -25,6 +25,10 @@ random sampling procedures.
 - |Fix| :class:`manifold.TSNE` now avoids numerical underflow issues during
   affinity matrix computation.
 
+- |Feature|, |Enhancement| :class:`decomposition.KernelPCA` now offers a new
+  `'randomized'` solver for faster fit.
+
+
```

Comment:
```suggestion
```
This `Changed models` section of the what's new document is reserved to models that might give different results compared to previous versions. Because this PR is a new feature, we should not add anything here.

Reply:
Indeed, it used to be the case in a previous iteration of this PR but this is no longer the case.